blint/cli.py

```diff
@@ -86,14 +99,18 @@ def main():
         exit()
     for dir in src_dir:
         if not os.path.exists(dir):
-            print(f"{src_dir} is an invalid file or directory!")
+            print(f"{dir} is an invalid file or directory!")
```

Comment:
I think `dir` is a builtin. So let's use some other variable.

Reply:
renamed